From f720b552be3eda6cfc5d5e4f3b90c3bfce3f5fdf Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Thu, 26 Sep 2013 16:07:15 +0800
Subject: [PATCH 254/310] ENGR00276661 CTS: fix network CTS test failure item

auto board don't support 3g mobile. Delete it in net attribute.
or it will cause cts.ConnectivityManagerTest#testGetAllNetworkInfo fail.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 .../overlay/frameworks/base/core/res/res/values/config.xml        | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml b/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
index 72b2cf0..d51d234 100755
--- a/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
@@ -71,14 +71,6 @@
     <!-- the 6th element indicates boot-time dependency-met value. -->
     <string-array translatable="false" name="networkAttributes">
         <item>"wifi,1,1,1,-1,true"</item>
-        <item>"mobile,0,0,0,-1,true"</item>
-        <item>"mobile_mms,2,0,3,60000,true"</item>
-        <item>"mobile_supl,3,0,3,60000,true"</item>
-        <item>"mobile_hipri,5,0,4,60000,true"</item>
-        <item>"mobile_fota,10,0,3,60000,true"</item>
-        <item>"mobile_ims,11,0,3,60000,true"</item>
-        <item>"mobile_cbs,12,0,3,60000,true"</item>
-        <item>"bluetooth,7,7,2,-1,true"</item>
         <item>"ethernet,9,9,2,-1,true"</item>
     </string-array>
 
-- 
1.8.0

